src/cloudai/workloads/ai_dynamo/ai_dynamo.sh

```
@@ -2,68 +2,74 @@
-GENAI_PERF_ARTIFACT_DIR="genai_perf_artifacts"
 
 function log()
 {
-  echo "[$(date --iso-8601=ns) $(hostname)]: $@"
+  echo -e "[$(date +%F\ %T) $(hostname)]: $*"
+}
+
+function min()
+{
+  echo "$(( $1 < $2 ? $1 : $2 ))"
 }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)